Mathlib/Analysis/FunctionalSpaces/SobolevInequality.lean

```diff
@@ -562,7 +560,15 @@ theorem eLpNorm_le_eLpNorm_fderiv_of_eq_inner  {u : E → F'}
       gcongr
       simp_rw [← mul_assoc]
       exact enorm_fderiv_norm_rpow_le (hu.differentiable le_rfl) h1γ
-      fun_prop
+      dsimp [enorm]
```

Comment:
I think this regression is unfortunate. Somebody should investigate; perhaps I'm nerd-sniped. Can you at least try to re-golf the remainder with fun_prop?

Reply:
Oh wait sorry, I accidentally committed my debugging of why `fun_prop` broke. But now `fun_prop` is fixed